spec/models/replacement_document_validation_request_spec.rb

```diff
@@ -101,11 +101,11 @@
           create(:replacement_document_validation_request, planning_application: planning_application)
         end
 
-        it "prevents a replacement_document_validation_request from being created" do
```

Comment:
This means there's now no test for `ensure_planning_application_not_validated!`. Can someone suggest a good alternative example of something that should be possible only before validation? @benbaumann95 @EGiataganas

Reply:
Other change validation requests should not be possible after validation